docs/restic.md

```diff
@@ -128,6 +128,10 @@ common encryption key for all restic repositories created by Velero. **This mean
 bucket can decrypt your restic backup data**. Make sure that you limit access to the restic bucket
 appropriately. We plan to implement full Velero backup encryption, including securing the restic encryption keys, in 
 a future release.
+- The current Velero/restic integration relies on using pod names to associate restic backups with their parents. If a pod is restarted, such as with a Deployment, 
+the next restic backup taken will be treated as a completely new backup, not an incremental one.
+- Restic processes each file in a single thread. This means that large files (such ones storing a database) will take a long time to scan for data deduplication, even if the actual 
```

Comment:
"such *as* ones storing..."

Instead of "processes each file" I'd maybe say "scans each file for changes" -- since if there are new data blocks, they're packed/written to the repo concurrently, it's just the scan that's single-threaded.